core/src/test/java/org/infinispan/persistence/support/AsyncStoreWithoutEvictionFunctionalTest.java

```diff
@@ -36,15 +38,10 @@
    private DefaultCacheManager dcm;
 
    private boolean segmented;
-   private String tmpDirectory;
-
-   @BeforeClass
-   protected void setUpTempDir() {
-      tmpDirectory = TestingUtil.tmpDirectory(this.getClass());
-   }
+   private File tmpDirectory;
 
    @AfterClass
-   protected void clearTempDir() {
+   public void clearTempDir() {
```

Comment:
Need `@AfterClass(alwaysRun = true)` as well to run after failures

Reply:
Sure

Reply:
Updated